dask/array_api/_elementwise_functions.py

```diff
@@ -0,0 +1,239 @@
+# Element-wise Functions
+# https://data-apis.org/array-api/latest/API_specification/elementwise_functions.html
+
+import numpy as np
+
+from ._dtypes import _integer_dtypes
+
+
+def abs(x, /):
+    return np.abs(x)
+
+
+def acos(x, /):
+    return np.arccos(x)
+
+
+def acosh(x, /):
+    return np.arccosh(x)
+
+
+def add(x1, x2, /):
+    return np.add(x1, x2)
+
+
+def asin(x, /):
+    return np.arcsin(x)
+
+
+def asinh(x, /):
+    return np.arcsinh(x)
+
+
+def atan(x, /):
+    return np.arctan(x)
+
+
+def atan2(x1, x2, /):
+    return np.arctan2(x1, x2)
+
+
+def atanh(x, /):
+    return np.arctanh(x)
+
+
+def bitwise_and(x1, x2, /):
+    return np.bitwise_and(x1, x2)
+
+
+def bitwise_invert(x, /):
+    return np.invert(x)
+
+
+def bitwise_left_shift(x1, x2, /):
+    return np.left_shift(x1, x2)
+
+
+def bitwise_or(x1, x2, /):
+    return np.bitwise_or(x1, x2)
+
+
+def bitwise_right_shift(x1, x2, /):
+    return np.right_shift(x1, x2)
+
+
+def bitwise_xor(x1, x2, /):
+    return np.bitwise_xor(x1, x2)
+
+
+def ceil(x, /):
+    if x.dtype in _integer_dtypes:
+        # Note: The return dtype of ceil is the same as the input
```

Comment:
FYI the "# Note" comments in the numpy.array_api implementation are there to carefully document the differences from the mainline NumPy.